SimGeneral/TrackingAnalysis/python/trackingParticleNumberOfLayersProducer_cff.py

```diff
@@ -0,0 +1,3 @@
+from SimGeneral.TrackingAnalysis.trackingParticleNumberOfLayersProducer_cfi import *
+from Configuration.StandardSequences.Eras import eras
+eras.fastSim.toModify(trackingParticleNumberOfLayersProducer, simHits=['famosSimHits:TrackerHits'])
```

Comment:
Thanks David, I didn't think of that. I'll try it and push an update if there are no obstacles.